src/main/java/homework3/ex1/page/Ex1HomePage.java

```diff
@@ -0,0 +1,111 @@
+package homework3.ex1.page;
+
+import homework3.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import java.util.List;
+
+public class Ex1HomePage extends BasePage {
+
+    private WebDriver driver;
```

Comment:
you don't need this field here because you have one protected in the parent. Remove